number/number_test.go

```diff
@@ -0,0 +1,72 @@
+package number
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/messagebird/go-rest-api/internal/mbtest"
+)
+
+func TestMain(m *testing.M) {
+	mbtest.EnableServer(m)
+}
+
+func TestSearch(t *testing.T) {
+	mbtest.WillReturnTestdata(t, "numberSearch.json", http.StatusOK)
+	client := mbtest.Client(t)
+
+	numLis, err := Search(client, "NL", &NumberListParams{Limit: 10, Features: []string{"sms", "voice"}, Type: []string{"mobile"}})
+	if err != nil {
+		t.Fatalf("unexpected error searching Numbers: %s", err)
+	}
+
+	if numLis.Limit > 100 {
```

Comment:
As you're operating on test data in this test, you can assert the exact value here.